lib/add_connection.dart

```diff
@@ -0,0 +1,271 @@
+/*
+ * This file is part of the Embla Flutter app
+ * Copyright (c) 2020-2022 Miðeind ehf. <mideind@mideind.is>
+ * Original author: Sveinbjorn Thordarson
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
It would be nice if the bottom of the header of the new Dart source files contained a brief description of the functionality contained therein.